tests/integration/components/settings/application-page-test.js

```diff
@@ -0,0 +1,25 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent('settings/application-page', 'Integration | Component | settings/application page', {
```

Comment:
@niranjan94 I am not getting what change has to be made. Do we have to create it same as the events? Where is the change needed?